index.bs

```diff
@@ -4090,6 +4090,13 @@ To <dfn export>run WebDriver BiDi preload scripts</dfn> given |environment setti
 
     1. Let |function declaration| be |preload script|'s <code>function declaration</code>.
 
+    1. Let |context id| be |preload script|'s <code>context</code>.
+
+    1. Let |context| be the result of [=trying=] to [=get a browsing context=]
+       with |context id|.
+
+    // TODO: Need to use |context| somewhere.
```

Comment:
You need to check something like `if |context| is not null and |context|`s [=active window=] is not |environment setting|'s [=relevant global object=], then continue'. But you want to move this check up to before we try to deserialize arguments, so we aren't expected to throw an error in the case where argument deserialization would fail but the script wouldn't run anyway.

Reply:
I am not familiar enough with the precision of the spec at this point, so I will put this on hold. If someone wants to continue from here, feel free to take over this PR.

@sadym-chromium FYI.